Makefile

```diff
@@ -71,6 +71,26 @@ GO ?= CGO_ENABLED=0 go
 # Allow setting of go build flags from the command line.
 GOFLAGS := 
 
+# Location to install dependencies to
+LOCALBIN ?= $(shell pwd)/bin
+$(LOCALBIN):
+	mkdir -p $(LOCALBIN)
+
+# Controller Gen for crds
+CONTROLLER_GEN ?= $(LOCALBIN)/controller-gen
+CONTROLLER_TOOLS_VERSION ?= v0.13.0
+
+controller-gen: $(CONTROLLER_GEN) ## Download controller-gen locally if necessary. If wrong version is installed, it will be overwritten.
+$(CONTROLLER_GEN): $(LOCALBIN)
+	test -s $(LOCALBIN)/controller-gen && $(LOCALBIN)/controller-gen --version | grep -q $(CONTROLLER_TOOLS_VERSION) || \
+	GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-tools/cmd/controller-gen@$(CONTROLLER_TOOLS_VERSION)
+
+.PHONY: codegen
+codegen: controller-gen  ## Generate ClusterRole, CRDs and DeepCopyObject.
+	$(CONTROLLER_GEN) crd paths="./api/..." output:crd:artifacts:config=config/operator/crd/bases
```

Comment:
We still need to get closure on this (i.e. `pkg/apis`)

Reply:
I'm wondering if we should create a separate output folder for the operator.
Maybe just use `config/crds`? 

Another option is `config/{core,operator}/{crds, rbac}`